py3status/modules/volume_status.py

```diff
@@ -157,10 +158,10 @@ def setup(self, parent):
 
     def get_volume(self):
         try:
-            perc = check_output(self.cmd + ["--get-volume"])
-        except CalledProcessError as cpe:
+            perc = self.command_output(self.cmd + ["--get-volume"])
+        except CommandError as ce:
             # pamixer throws error on zero percent. see #1135
-            perc = cpe.output
+            perc = ce.output
 
         perc = perc.decode().strip()
```

Comment:
It breaks. We need to remove this `.decode()` too.

Reply:
Right, fixed.